src/elements/Progressbar/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import style from './style.scss';
+
+const cx = classNames.bind(style);
+
+class Progressbar extends React.Component {
+  constructor (props) {
+    super(props);
+    const { percentage, showPercentageNum, className } = props;
+
+    this.percentage = percentage || 0;
+    this.showPercentageNum = showPercentageNum;
+    this.className = className;
+  }
+
+  getBarClass () {
+    return cx(
+      'progress-bar',
+      this.className);
+  }
+
+  render () {
+    const progressBarWidth = this.percentage > 100 ? 100 : this.percentage;
+    const percentageLabel = this.showPercentageNum ? `${this.percentage}%` : null;
+
+    return (
+      <div>
+        <div className={cx('progress-bar-container')}>
+          <div style={{ width: `${progressBarWidth}%` }} className={this.getBarClass()}>
+            {percentageLabel}
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+Progressbar.propTypes = {
+  className: PropTypes.string,
+  percentage: PropTypes.number,
+  showPercentageNum: PropTypes.bool,
+};
+
+Progressbar.defaultProps = {
+  percentage: 20,
+};
+
+export { Progressbar as default };
```

Comment:
you just can do `export default Progressbar`. Don't need to have `Progressbar as default`. This is useful when you export multiple component or same component with different context.